packages/adapter-d1/README.md

```diff
@@ -1,3 +1,79 @@
 # Prisma driver adapter for Cloudflare D1
 
-**INTERNAL PACKAGE, DO NOT USE**
+Prisma driver adapter for [Cloudflare's Serverless Driver D1](https://developers.cloudflare.com/d1/). Refer to the [announcement blog post](https://prisma.io/cloudflare-d1) and our [docs](https://www.prisma.io/docs/guides/database/cloudflare-d1) for more details.
+
+> **Note**: Support for Cloudflare D1 is available in [Early Access](https://www.prisma.io/docs/about/prisma/releases#early-access) from Prisma versions [TODO](https://github.com/prisma/prisma/releases/tag/TODO) and later.
```

Comment:
We do not document that in the README.

Reply:
We have these in our other adapter readmes, should we not be?

Reply:
Those only had Preview, not Early Access. When we go Preview, we can also document that. Until then remove or comment out.